### PR TITLE
[TACHYON-583] Update config docs with available remote block reader o…

### PR DESCRIPTION
**Pull request by miasantreble**
…ptions

### CODE REVIEW

File: docs/Configuration-Settings.md [hunk 1]
```diff
@@ -377,7 +377,9 @@ The user configuration specifies values regarding file system access.
 <tr>
   <td>tachyon.user.remote.block.reader.class</td>
   <td>tachyon.client.tcp.TCPRemoteBlockReader</td>
-  <td>Selects networking stack to run the client with.</td>
+  <td>Selects networking stack to run the client with. Valid options are
```
Comment (calvinjia) [inline docs/Configuration-Settings.md]: Could you update the default to netty? I just opened #1129

File: docs/Configuration-Settings.md [hunk 2]
```diff
@@ -377,7 +377,9 @@ The user configuration specifies values regarding file system access.
...
+    tachyon.client.tcp.TCPRemoteBlockReader (read remote data using tcp)
+    and tachyon.client.netty.NettyRemoteBlockReader(read remote data using netty).</td>
```
Comment (calvinjia) [inline docs/Configuration-Settings.md]: Should there be a space before the `(`?

Comment (miasantreble) [inline docs/Configuration-Settings.md]: Done

**Comment by apc999** [comment]
+1 LGTM

**Comment by calvinjia** [comment]
@miasantreble Could you revert the change to of the default, I think it probably makes more sense to put it in #1129

**Comment by miasantreble** [comment]
@calvinjia default reader reverted

**Comment by calvinjia** [comment]
+1 LGTM

**Pull request merged**